spec/models/report_spec.rb

```diff
@@ -58,6 +58,47 @@
     end
   end
 
+  describe '#keyword_search' do
+    let!(:searchable_report){ create(:report, name:'John Doe', phone: '5103874543', address: '135 Main St.', status: 'pending', nature: 'urgent') }
```

Comment:
Maybe just a comment here about why we need to use `let!` instead of just `let`. Something like:

> using `let!` because we need the report to exist before searching and it is only referenced in the comparison